packages/blade/src/components/Typography/BaseText/StyledBaseText.native.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components/native';
+import getBaseTextStyles from './getBaseTextStyles';
+import type { BaseTextProps } from './BaseText.d';
+
+const StyledBaseText = styled.Text<BaseTextProps>(
```

Comment:
For some cases can't we just detect platform and based on that conditionally change logic instead of copy pasting the similar code for native and web? 

like `Platform.isNative ? styled.Text : styled.div`

Reply:
> BaseText renders weirdly on iOS.
> 
> <img alt="Screenshot 2022-04-21 at 4 45 59 PM" width="403" src="https://user-images.githubusercontent.com/24487274/164446750-009de809-cb15-4fae-83ac-5e94fc89fe65.png">

as we has discussed today, we won't control this as `BaseText` is meant to be flexible but with tokens constraints in place